tests/publiccloud/ipa.pm

```diff
@@ -143,4 +183,9 @@ This is B<only for azure> and used to create the service account file.
 
 This is B<only for azure> and used to create the service account file.
 
+=head2 PUBLIC_CLOUD_PERF_DB_URI
+
+If this variable is set, the bootup timings get stored inside the influx
```

Comment:
I would add something about expected format of URI

Reply:
Ok, I will add the URL I'm using as an example.